### PR TITLE
LPS-133507 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-135507

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Just start reviewing

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+		task ("Configue the password and the reminder") {
+			User.resetPassword();
+
+			User.answerPasswordReminderLocalized(
+				localizedAnswer = "Respuesta",
+				localizedSaveButton = "Guardar");
+		}
+
+		task ("View default welcome content page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase]: View the Home page is localized

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+				key_fragmentName = "heading",
+				locator1 = "Fragment#FRAGMENT_BASIC_COMPONENT",
+				value1 = "Bienvenido a Liferay");
+
+			AssertVisible(
+				id = "image-square",
+				key_image = "tree.png",
+				key_imageDescription = "Responsive Image",
+				locator1 = "Fragment#CONTRIBUTED_FRAGMENT_EDITABLE_FIELD_IMAGE_PROPERTIES");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase]: Only keep these two steps, then move to Locale.RestrictDefaultLocale

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update the case according to comments?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase (hunk 2 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase]: > Only keep these two steps, then move to Locale.RestrictDefaultLocale

Hi @Tim-Cao, I tried to put these two steps on the file above like you said but it's not working. Could you show me how it supposed to be? Thanks in advance.

I think it's just going to work if I set the basic configuration to Spanish manually and then run the Locale.RestrictDefaultLocale, but the description says to automate all process.

Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase]: Did you add company.default.locale=es_ES to your portal-ext.properties ?

Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentePagesWithSetupWizard.testcase]: I didn't, it's the first time that I'm doing a test with the basic configuration. I'll add and try to do it again.

**Comment by ozysouza** [comment]
ci:reopen

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update the case according to comments?

Hi @Tim-Cao, PR updated and I also included the Case 3.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase [hunk 3]
```diff
@@ -119,6 +119,50 @@ definition {
 		LanguagePortlet.viewDisplayTemplateShortText(languageKey = "hu");
 	}
 
+	@description = "This is a test for LPS-134072. View the Home page appending with available languages."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase]: Move the case into FriendlyURL#ViewLocaleChangeThroughFriendlyURL

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase [hunk 4]
```diff
@@ -119,6 +119,50 @@ definition {
...
+	@priority = "4"
+	test ViewHomePageAppendingURLWithAvailableLanguages {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase:124]: ViewHomePageInDifferentUserLanguages

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase [hunk 5]
```diff
@@ -119,6 +119,50 @@ definition {
...
+		task ("View the Home page as default language") {
+			AssertTextEquals(
+				key_fragmentName = "heading",
+				locator1 = "Fragment#FRAGMENT_BASIC_COMPONENT",
+				value1 = "Welcome to Liferay");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase]: Add step to view image shown

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase [hunk 6]
```diff
@@ -119,6 +119,50 @@ definition {
...
+		task ("View the Home page in Chinese") {
+			AssertTextEquals(
+				key_fragmentName = "heading",
+				locator1 = "Fragment#FRAGMENT_BASIC_COMPONENT",
+				value1 = "欢迎来到Liferay");
+		}
+
+		task ("Append locale URL to Portuguese") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "pt/web/guest/home");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase]: Add step to view image shown

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase [hunk 7]
```diff
@@ -119,6 +119,50 @@ definition {
...
+
+		task ("View the Home page in Portuguese") {
+			AssertTextEquals(
+				key_fragmentName = "heading",
+				locator1 = "Fragment#FRAGMENT_BASIC_COMPONENT",
+				value1 = "Bem Vindo ao Liferay");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase]: Add step to view image shown

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase [hunk 8]
```diff
@@ -119,6 +119,50 @@ definition {
...
+		task ("Append locale URL to French") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "fr/web/guest/home");
+		}
+
+		task ("View the Home page in French") {
+			AssertTextEquals(
+				key_fragmentName = "heading",
+				locator1 = "Fragment#FRAGMENT_BASIC_COMPONENT",
+				value1 = "Bienvenue à Liferay");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/languageselector/LocalizationUseCase.testcase]: Add step to view image shown

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update your case according to Case 3 steps defined in https://issues.liferay.com/browse/LPS-134848? Thanks.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, I sent the Case 2 here, see changes
https://github.com/liferay-echo/liferay-portal/compare/3cd8e59613573b5baf16610c45aec8d1236f9621...df96c9117dce6381589917db3628739a8bf906e9

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update your case according to Case 3 steps defined in https://issues.liferay.com/browse/LPS-134848? Thanks.

Hi @Tim-Cao, PR updated following your comments.

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/8ce001e1d71ee6300df574b5f352d53251d44dcd...7cb88a81b5d57af87d6e23aaaeacc95916b22ffa

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**